charts/posthog/values.yaml

```diff
@@ -445,6 +445,16 @@ redis:
 
       # -- Persistent Volume size.
       size: 5Gi
+    extraFlags:
+      ## The maxmemory configuration directive is used in order to configure Redis to use a specified
+      ## amount of memory for the data set. Setting maxmemory to zero results into no memory limits
+      ## see https://redis.io/topics/lru-cache for more details
+      - "--maxmemory 4500mb"
```

Comment:
Didn't we use 200MB earlier? Otherwise it will be a quite costly service to run. Do we really need that much memory? 🤔

Reply:
We used 200MB for hobby + docker-compose. Here I set it to 4500MB based on the default for Redis disk we ask is 5GB

Reply:
yeah exactly I just realized that too 🤦‍♀️  why do we need 5GB of disk for Redis, what is it used for?
Will set it to 200MB like everywhere else unless we think it should be something else.

Reply:
I do think 200mb is too low, however. This will probably blow up a lot of instances like N****ta and V*****ta that use the default Redis.

Worth asking them how much they use? One of them we have k8s access anyway. 

Also we should not merge this before the Snowflake stuff and whatever else needs changing.

Reply:
I can tell you V. has 3gb available on Redis 360mb

Reply:
Instance status shows these

Reply:
that's cool, added the maxmemory info there too as that's relevant too now https://github.com/PostHog/posthog/pull/9660

Reply:
Do we want to set it to be relatively small like everything else (the current chart is rather small default based & I'd like to keep it that way so small enterprises can get off the ground more easily) and make it major upgrade asking bigger users like V. to pick their limit?

Reply:
Well my bigger concern is breaking existing users

Reply:
If we can minimize the number of users we break on self-hosted it would be great.

Let's maybe do 400mb?

Reply:
sounds good to me